3-dense.ipynb

```diff
@@ -467,14 +467,14 @@
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 3
+    "version": 2
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.6.10"
+   "pygments_lexer": "ipython2",
+   "version": "2.7.5"
```

Comment:
Why did this switch from python3 to python2?

Reply:
shouldn't we update to python3 at this point? does anything in this tutorial require python2?

Reply:
I definitely think we should be using python3 and we were. I made the comment because you seem to be using python2.